idl/partition_checksum.idl.hh

```diff
@@ -115,6 +126,11 @@ struct node_ops_cmd_request {
     std::unordered_map<gms::inet_address, std::list<dht::token>> bootstrap_nodes;
     // Optional field, list uuids of tables being repaired, set by repair cmd
     std::list<table_id> repair_tables;
+
+    // v2:
+    // Optional field, maps node index to (host_id, inet_address)
+    // gms:inet_address values above will contain the node index as a raw ipv4 address
+    std::unordered_map<locator::node::idx_type, locator::host_id_and_endpoint> nodes_dict;
```

Comment:
Later I understood that the node indexes are into this map here that is also transferred on the wire. So that is fine. Still think this is quite hacky.

Reply:
yeah, it's a way to extend the rpc in a backward compatible way without requiring a new set of verbs and handlers.